ssntp/certs/certs.go

```diff
@@ -0,0 +1,240 @@
+//
+// Copyright (c) 2016 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Initial implementation based on
+//    golang/src/pkg/crypto/tls/generate_cert.go
+//
+// which is:
+//
+//    Copyright 2009 The Go Authors. All rights reserved.
+//    Use of this source code is governed by a BSD-style
+//    license that can be found in the golang LICENSE file.
+//
+
+package certs
```

Comment:
I think we're going to need some unit tests for all this stuff, particularly as we are adding a new package.